run.sh

```diff
@@ -1,21 +1,75 @@
 #!/bin/sh
-parse_yaml() {
-   local prefix=$2
-   local s='[[:space:]]*' w='[a-zA-Z0-9_]*' fs=$(echo @|tr @ '\034')
-   sed -ne "s|^\($s\)\($w\)$s:$s\"\(.*\)\"$s\$|\1$fs\2$fs\3|p" \
-        -e "s|^\($s\)\($w\)$s:$s\(.*\)$s\$|\1$fs\2$fs\3|p"  $1 |
-   awk -F$fs '{
-      indent = length($1)/2;
-      vname[indent] = $2;
-      for (i in vname) {if (i > indent) {delete vname[i]}}
-      if (length($3) > 0) {
-         vn=""; for (i=0; i<indent; i++) {vn=(vn)(vname[i])("_")}
-         printf("%s%s%s=\"%s\"\n", "'$prefix'",vn, $2, $3);
-      }
-   }'
+
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+
+   export "$KEY"="$VALUE"
+done
+
+frontend(){
+   echo "Running frontend $framework"
+   if [[ $framework == "react" ]]
+   then
+      cd client && export PORT=3001 && npm start
+   fi
+   if [[ $framework == "dash" ]]
+   then
+      python run_dash_frontend.py $yaml
+   fi
+}
+
+server(){
+    echo "Running summarization model $yaml"
+    is_elastic=$(echo $(python parse_yaml.py $yaml models_search)| grep -c "ElasticBERT") 
+    if (($is_elastic >= 1 ))
+    then
+		is_elastic_service_running=$(echo $(curl http://localhost:9200/_cluster/health) | grep -c "elasticsearch")
```

Comment:
Will elasticsearch always default to 9200? Maybe we can have something in our README where we list all default ports we expect to be free/BLAZE to use